daemon/daemon.go

```diff
@@ -101,6 +109,13 @@ func (d *Daemon) Run() error {
 		SystemMgr:    systemMgr,
 		ImageMgr:     imageMgr,
 		VolumeMgr:    volumeMgr,
+		NetworkMgr:   networkMgr,
+	}
+
+	// init bridge network
+	err = bridge.New(ctx, d.config.NetworkConfg.BridgeConfig, networkMgr)
```

Comment:
I think this could be encapsulated in the network manager initialization.

Reply:
Agree, current it is miss, next we will add more network modules, it should be encapsulated in network manager.